python/doc/examples/meta_modeling/polynomial_chaos_metamodel/plot_chaos_lognormal.py

```diff
@@ -0,0 +1,293 @@
+lower_Bound = dist_X.computeQuantile(0.01, False)
+upper_Bound = dist_X.computeQuantile(0.01, True)
+degreeMax = 30
+
+as_Algo = ot.AdaptiveStieltjesAlgorithm(dist_X)
+sample_Coeff = ot.Sample(0, 3)
+for i in range(degreeMax):
+    coeff = as_Algo.getRecurrenceCoefficients(i)
+    sample_Coeff.add(coeff)
+
+# %%
+# We build a set of polynomials with increasing degrees.
+# For each polynomial, we display its value at :math:`x = 0`.
+#
+# We anticipate that the vicinity of :math:`x = 0` will cause difficulties
+# for the surrogate model of :math:`\model`, since
+# :math:`\lim_{x \rightarrow 0} \model(x) = +\infty`.
+# Given that the polynomials take *reasonable values* in the neighborhood of :math:`x = 0`,
+# we expect the polynomial surrogate model to require
+# very large linear coefficients in order to reproduce high values near :math:`x = 0`,
+# which may lead to numerical cancellation issues.
+#
+# We explore polynomial degrees up to 40.
+deg_list = [5, 10, 20, 30, 40]
+g = ot.Graph('Polynomials orthonormal  with respect to the LogNormal distribution', 'x', 'y', True)
+leg = ot.Description(0)
+for indice, deg in enumerate(deg_list):
+    coeff_Poly_Deg = sample_Coeff[0:deg]
+    poly_Deg = ot.OrthogonalUniVariatePolynomial(coeff_Poly_Deg)
+    g.add(poly_Deg.draw(lower_Bound[0], upper_Bound[0], 1024))
+    leg.add(r'$d = $' + str(deg))
+    print('degree = ', deg, 'polynomial(0) = ', poly_Deg(0))
+
+g.setLegends(leg)
+g.setLegendPosition('topleft')
+view = otv.View(g)
+
+# %%
+# We observe that the polynomials up to degree 40:
+#
+# - are computed in a stable manner, as indicated by their smooth graphs,
+# - take reasonable values on the interval :math:`[0, 25]`.
+
+# %%
+# Case 1: PCE using orthonormal polynomial family w.r.t. the LogNormal distribution
+# ---------------------------------------------------------------------------------
+#
+# We define :math:`\model`, the input random variable :math:`X` and the output random variable :math:`Y = \model(X)`.
+input_Dim = dist_X.getDimension()
+input_RV = ot.RandomVector(dist_X)
+
+g = ot.SymbolicFunction('x', '1/(x)')
+
+output_RV = ot.CompositeRandomVector(g, input_RV)
+
+# %%
+# We create a training sample.
+sample_Size = 1000
+input_Train = dist_X.getSample(sample_Size)
+output_Train = g(input_Train)
+print('max sample = ', input_Train.getMax()[0])
+
+
+# %%
+# We now construct the polynomial chaos surrogate model :math:`\metaModel`
+# using the polynomials up to degree 5. Note that the coefficients in the linear combination of
+# the polynomials are large.
+basis_LN = ot.OrthogonalProductPolynomialFactory([ot.StandardDistributionPolynomialFactory(ot.LogNormal())])
+basis_Size_LN = 6
+chaos_Algo_LN = ot.LeastSquaresExpansion(input_Train, output_Train, dist_X, basis_LN, basis_Size_LN)
+chaos_Algo_LN.run()
+result_LN = chaos_Algo_LN.getResult()
+meta_Model_LN = result_LN.getMetaModel()
+print('meta_Model_LN = ', meta_Model_LN)
+print('coeff = ', result_LN.getCoefficients())
```

Comment:
This produces a not-so-pretty truncated output:

<img width="661" height="475" alt="image" src="https://github.com/user-attachments/assets/a05c0d25-6b99-4b1d-8113-95e3c1277404" />

I suggest to let the pretty-print make its job:
```suggestion
result_LN 
```

This will produce:

FunctionalChaosResult
<ul>
  <li>input dimension: 1</li>
  <li>output dimension: 1</li>
  <li>distribution dimension: 1</li>
  <li>transformation: 1 -> 1</li>
  <li>inverse transformation: 1 -> 1</li>
  <li>orthogonal basis dimension: 1</li>
  <li>indices size: 6</li>
</ul>
<table>
  <tr>
    <th>Index</th>
    <th>Multi-index</th>
    <th>Coeff.</th>
  </tr>
  <tr>
    <th>0</th>
    <td>[0]</td>
    <td>-23.67553</td>
  </tr>
  <tr>
    <th>1</th>
    <td>[1]</td>
    <td>-3651.218</td>
  </tr>
  <tr>
    <th>2</th>
    <td>[2]</td>
    <td>-111611.1</td>
  </tr>
  <tr>
    <th>3</th>
    <td>[3]</td>
    <td>-658095</td>
  </tr>
  <tr>
    <th>4</th>
    <td>[4]</td>
    <td>-798496.9</td>
  </tr>
  <tr>
    <th>5</th>
    <td>[5]</td>
    <td>-238352</td>
  </tr>
</table>

Reply:
yes!